.github/workflows/push-to-testnet-draft.yml

```diff
@@ -9,72 +9,22 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        sbc: [raspi, rockpi, rak, finestra, sensecap, og, controllino, pisces, cotx]
+        variant: [nebra-indoor1, nebra-outdoor1, nebra-indoor2, nebra-light1, nebra-light2, nebra-light3]
     steps:
       - name: Checkout
         uses: actions/checkout@v2
+        with:
+          ref: master
```

Comment:
Don't think this should be here. Because it's a workflow dispatch action the whole point is we can run it on arbitrary branches (the production one should only be production because that's specific but this one doesn't need to be only master IMO)

Reply:
Yea, I really tried my chance on this. Didn't know exact necessity and triggered it to see effects and learn more about it :smile: